garage/tf/algos/npo.py

```diff
@@ -33,14 +33,33 @@ class NPO(BatchPolopt):
     """
     Natural Policy Gradient Optimization.
 
-    Attributes:
-        name(str): The name of the algorithm.
+    Args:
```

Comment:
please also note which params are optional

Reply:
on second thought, the documentation generator makes it clear what's optional, and these notations are just noise. please remove.

i apologize for changing my mind.

Reply:
Which documentation generator can make it clear about optional parameters? I tried with pydoc and pdoc but they don’t specify optional parameters.

Reply:
For instance, in akro the module documentation looks like this:
https://akro.readthedocs.io/en/latest/_apidoc/akro.html#module-akro.box

You can see the function signature with kwargs, so it's obvious the kwargs are optional and the args are not.